site/source/tutorials/1030_routing/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: tutorial
+title: Routing
+overview: In this tutorial, you will learn how to use Dojo 2's declarative routing within your application.
+---
+
+{% section 'first' %}
+
+# Application routing
+
+## Overview
+
+Dojo 2's routing is a powerful set of tools to support declarative routing using a high order component, an `Outlet` and a component that creates links with a `href` generated from an `outlet` name.
+
+In this tutorial, we will start with a basic application with no routing. We will use Dojo 2's declarative routing to configure some routes, create `outlets` from our existing widgets and use the `Link` component to create links for the application outlets.
+
+## Prerequisites
+You can [download](../assets/1030_routing-initial.zip) the demo project and run `npm install` to get started.
+
+The `@dojo/cli` command line tool should be installed globally. Refer to the [Dojo 2 local installation](../000_local_installation/) article for more information.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+
+{% section %}
+
+## Configuring the router
+
+{% task 'Create the routing configuration' %}
+
+Talk about how to and creating the routing configuration.
+
+{% instruction 'Add the routing configuration to `main.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' lines:9-31 %}
```

Comment:
we should break this down and talk about the different types of routing patterns that are demonstrated (e.g default routes, parameterized routes, etc)